src/components/10-atoms/input-text/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
```

Comment:
You can use our new changelog template to format this file if you want.

Reply:
We should do that in an extra-round imho